### ISSUE TITLE
Save the adjustment of the lighting to LLE file save?

### ISSUE THREAD

**Issue by SameSkeletonDifferentCloset** [mdotstrange/NightmarePuppeteerPublic#287]
Would that be a small thing to add?

**Comment by mdotstrange** [comment]
Would this be the adjustment to the lighting using the L key? If so I can def make it so it is saved/loaded with scenes.

**Comment by SameSkeletonDifferentCloset** [comment]
> Would this be the adjustment to the lighting using the L key? If so I can def make it so it is saved/loaded with scenes.

Yes that's it! It's one damn fine feature you added by the way. Thanks M!
